### PR TITLE
Vue

### PR DESCRIPTION
**Pull request by matyunya**

### CODE REVIEW

File: utils/vue.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import Vue from 'vue';
+
+const ellxify = Component => class {
+  constructor(props, { initState }) {
+    this.value = initState;
+    this.target = document.createElement('div');
```
Comment (dmaevsky) [inline utils/vue.js]: not used

File: utils/vue.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    this.instance.$watch('value', (v) => this.emit(v));
+  }
+
+  stale() {
+    this.instance.$props.stale = true;
+  }
+
+  update(props = {}) {
+    for (let key in props) {
+      if (key === '__EllxMeta__') continue;
```
Comment (dmaevsky) [inline utils/vue.js]: why special treatment ?

Comment (matyunya) [reply utils/vue.js]: should we ever be passing it to components?

File: utils/vue.js (hunk 1 above)
Comment (matyunya) [reply utils/vue.js]: thx!

File: utils/vue.js (hunk 2 above)
Comment (dmaevsky) [reply utils/vue.js]: There's no obvious harm. If we ever encounter a situation where this is undesirable I would prefer just to make it non-enumerable in ellxify.

Comment (matyunya) [reply utils/vue.js]: removed

**Pull request merged**